commit f4598f0351559f1a4176d7ce0154423d98bcfb0d
Author: Noah Goldstein <goldstein.w.n@gmail.com>
Date:   Wed Jun 29 16:07:04 2022 -0700

    x86: Add definition for __wmemset_chk AVX2 RTM in ifunc impl list
    
    This was simply missing and meant we weren't testing it properly.
    
    (cherry picked from commit 2a1099020cdc1e4c9c928156aa85c8cf9d540291)

diff --git a/sysdeps/x86_64/multiarch/ifunc-impl-list.c b/sysdeps/x86_64/multiarch/ifunc-impl-list.c
index 043821278fdb6d8f..8d649e263eb24b8a 100644
--- a/sysdeps/x86_64/multiarch/ifunc-impl-list.c
+++ b/sysdeps/x86_64/multiarch/ifunc-impl-list.c
@@ -1032,6 +1032,10 @@ __libc_ifunc_impl_list (const char *name, struct libc_ifunc_impl *array,
 	      IFUNC_IMPL_ADD (array, i, __wmemset_chk,
 			      CPU_FEATURE_USABLE (AVX2),
 			      __wmemset_chk_avx2_unaligned)
+	      IFUNC_IMPL_ADD (array, i, __wmemset_chk,
+			      (CPU_FEATURE_USABLE (AVX2)
+			       && CPU_FEATURE_USABLE (RTM)),
+			      __wmemset_chk_avx2_unaligned_rtm)
 	      IFUNC_IMPL_ADD (array, i, __wmemset_chk,
 			      (CPU_FEATURE_USABLE (AVX512VL)
 			       && CPU_FEATURE_USABLE (AVX512BW)
